docs/backend/notification.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Notifications
+---
+
+ # Notifications
+
+ Since notifications are run asynchronously, we'll want to make sure jobs are running: `rake jobs:work`. If that's not running, you won't receive any notifications. You might need to create another account to pass notifications back and forth if you're doing this all through the UI.
```

Comment:
For consistency with the README and the start app doc, we might want to change the command to `bin/rails jobs:work`